### ISSUE TITLE
DockerComposeCompositeService.Remove does nothing

### ISSUE THREAD

**Issue by Mario-Hofstaetter** [mariotoffia/FluentDocker#113]
TL;DR:
**What is the use case for `DockerComposeCompositeService.Remove(true)` ?**

What I wanted to achive was... remove stopped (or running) containers after using a compose config.

----

FluentDocker `2.7.2`
I tried the following:

```csharp
Environment.SetEnvironmentVariable("DOCKER_HOST", "ax-docker02:2375");
var composition = new Builder()
                    .UseContainer()
                    .UseCompose()
                    .FromFile("testcase1.yml")
                    .Build();

composition.Start();
composition.Remove(force:true);
```
and noticed that `Remove(force:true);` doesn't do anything.
`DockerComposeCompositeService.Remove()` is not documented in `README.md` but the expected behavior was something like `docker-compose rm -sf` (stop containers and remove them).

Trying to understand and maybe debug, I checkout out the source of release git tag `2.7.2`.
Which leads to `DockerComposeCompositeService.cs`, 150:
```csharp
    public override void Remove(bool force = false)
    {
      State = ServiceRunningState.Removing;
      State = ServiceRunningState.Removed;
    }
```
which does nothing?

Since there are no source comments, could use please elaborate on this class?
This class also has a `[Experimental(TargetVersion = "3.0.0")]` Attribute

The method 
```csharp
public void Remove(bool force = false)
{
  foreach (var service in Services)
  {
    service.Remove(force);
  }
}
```
in `BuilderCompositeService.cs` looks more like what I expected does not seem to be used here.

[EDIT] So after writing #112 , I've found the following solution to above task:

```csharp
composition.Containers.ToList().ForEach(c => c.Remove());
```

But this raises a new problem: This will delete **Running** services (containers), even without specifying `force: true`, because the container will always be stopped.

Greetings

**Comment by mariotoffia** [comment]
Hi and thanks again for bugr reporting.

1) ```Remove()``` on the ```DockerCompositeService``` needs to be implemented as ```Stop```.
2) The _force: true_ is only valid for cases when it has some meaning such as in ```DockerHostService``` where _force: true_ will make sure that the _-f_ flag is added to the _docker-machine_ command.
3) The ```Experimental``` attribute are as state in the documentation :)
```cs
/// <summary>
  /// Specifies that the implementation is experimental and is subject to change.
  /// </summary>
  [AttributeUsage(AttributeTargets.All)]
  public sealed class ExperimentalAttribute : Attribute {

    /// <summary>
    /// Current target version when this is to be released.
    /// </summary>
    public string TargetVersion { get; set; }
}
```
In this case is is the milestone _3.0.0_. 

The behaviour of this service is not complete, hence the ```Experimental``` attribute. When finished it will adhere to same pattern as the ```ContainerService``` (& builder).

I haven't looked at your PR yet and will comment it separately.

Cheers,
 Mario :)

**Comment by mariotoffia** [comment]
@Mario-Hofstaetter If you have the time, you can test that the current implemntation is correct - I haven't written any unit test as of yet for this implementation.

Cheers,
 Mario

**Comment by mariotoffia** [comment]
Wrote unittest and they pass for this (and some other scenarios) - I'm closing this Issue now and it will be part of the next release.
